src/components/Popover/_theme.scss

```
@@ -11,6 +11,8 @@ $color: var(--rui-Popover__color);
 $background-color: var(--rui-Popover__background-color);
 $box-shadow: var(--rui-Popover__box-shadow);
 
-$arrow-width: 1rem;
-$arrow-height: math.div($arrow-width, 2); // 1.
+$arrow-safe-rendering-overlap: 1px;
+$arrow-gap: 1px;
+$arrow-width: calc(1rem + #{$arrow-safe-rendering-overlap + $arrow-gap});
+$arrow-height: calc(math.div($arrow-width, 2)); // 1.
```

The CSS division syntax does not work here?

```suggestion
$arrow-height: calc(#{$arrow-width} / 2); // 1.
```